standard/expressions.md

```diff
@@ -72,11 +72,14 @@ Static binding takes place at compile-time, whereas dynamic binding takes place
 >
 > <!-- Example: {template:"standalone-console", name:"BindingTime", expectedOutput:["5","5","5"]} -->
 > ```csharp
-> object o = 5;
-> dynamic d = 5;
-> Console.WriteLine(5); // static binding to Console.WriteLine(int)
-> Console.WriteLine(o); // static binding to Console.WriteLine(object)
-> Console.WriteLine(d); // dynamic binding to Console.WriteLine(int)
+> static void M()
```

Comment:
Do we need this declaration? If we used `code-in-main` as the template, could we basically leave the example as it was?

Reply:
Agreed; reverted.